scim_rails.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "~> 2.4"
   s.add_dependency "rails", ">= 5.0", "< 6.1"
-  s.add_runtime_dependency "jwt", "~> 1.5.1"
+  s.add_runtime_dependency "jwt", "~> 2.2"
```

Comment:
Why not `>= 1.5`? Is there an incompatibility?
That way people aren’t also forced to upgrade JWT to upgrade this SCIM gem.

Reply:
Great point...corrected. Thanks!